pages/validators/preparing_for_genesis.md

```diff
@@ -146,16 +133,16 @@ You can also use the below optional flags:
 ```
 
 ```bash
-dydxprotocold gentx $DYDX_KEY_NAME 50000000000dv4tnt --moniker $DYDX_MONIKER --keyring-backend test --chain-id $CHAIN_ID --home=$HOME_TESTNET_3
+dydxprotocold gentx $DYDX_KEY_NAME 50000000000dv4tnt --moniker $DYDX_MONIKER --keyring-backend test --chain-id $CHAIN_ID --home=$HOME_TESTNET_4
```

Comment:
I was planning on giving each validator 1million whole coin so 1e24 `adv4tnt`, so here they should probably run `gentx` with `5e23 adv4tnt`

Reply:
done!

I did the following:
1. update the `dydxprotocold add-genesis-account $MY_VALIDATOR_ADDRESS 1000000000000000000000000adv4tnt --home=$HOME_TESTNET_4`
2. update the `dydxprotocold gentx $DYDX_KEY_NAME 500000000000000000000000adv4tnt --moniker $DYDX_MONIKER --keyring-backend test --chain-id $CHAIN_ID --home=$HOME_TESTNET_4`

Reply:
Sg!